.github/workflows/tests.yml

```diff
@@ -0,0 +1,47 @@
+name: Tests
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+        - 3.6
+        - 3.7
+        - 3.8
+        - 3.9
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip wheel setuptools tox
+      - name: Run tox targets for ${{ matrix.python-version }}
+        run: |
+          ENV_PREFIX=$(tr -C -d "0-9" <<< "${{ matrix.python-version }}")
```

Comment:
Switching to GitHub Actions might be a good opportunity to run the tests on Windows too.  But this line isn't compatible with Windows.

![image](https://user-images.githubusercontent.com/152008/96522697-80fc1b00-1263-11eb-881f-4d0836f79fcc.png)

Source: https://github.com/michael-k/daphne/actions/runs/316525784

Reply:
A possible solution might be [tox-gh-actions](https://pypi.org/project/tox-gh-actions/), see this commit: https://github.com/michael-k/daphne/commit/49e85f012c03a42d245edecb4e1dbcd14f9e449d.

Tests fail if there are no wheels for twisted, ie. for Python 3.8+ (probably solved in next release, see https://github.com/twisted/twisted/blob/trunk/.github/workflows/wheels.yaml) and for twisted 18.7.0.

Reply:
Let's bump this to a separate PR. (But yes... 👍)

Reply:
I moved away from tox-gh-actions in my code since I disliked having an extra set of config to map python versinos to tasks that could rot. Tbh I could turn my shell one liner into a mini tox plugin, or I think it would be a good feature for tox itself e.g. `tox -e pycurrent` could run environments only for the current python version...

Reply:
> I moved away from tox-gh-actions in my code since I disliked having an extra set of config to map python versinos to tasks that could rot.

Yeah, same here.

> would be a good feature for tox itself

Absolutely!

Reply:
Btw. over at django-filter I've used Ryan P Kilby's [tox-factor](https://pypi.org/project/tox-factor/) and `python -m tox -f py$(python --version 2>&1 | cut -c 8,10)` which does seem to work on windows.

Reply:
Great find! Let's try that here?

Reply:
I also created an issue there to make it possible to use without the shell invocation: https://github.com/rpkilby/tox-factor/issues/14

Reply:
> Let's try that here?

I will, but (as Carlton suggested) in a separate PR.